src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -14518,15 +14542,18 @@ void Player_Action_19(Player* this, PlayState* play) {
     if (this->av1.actionVar1 == 0) {
         sUpperBodyIsBusy = Player_UpdateUpperBody(this, play);
         if ((Player_UpperAction_3 == this->upperActionFunc) ||
-            (func_808331FC(play, this, &this->skelAnimeUpper, 4.0f) > 0)) {
+            (Player_TryActionInterrupt(play, this, &this->skelAnimeUpper, 4.0f) >= PLAYER_INTERRUPT_MOVE)) {
             Player_SetAction(play, this, Player_Action_2, 1);
         }
     } else {
-        s32 temp_v0;
+        s32 interruptResult;
```

Comment:
I guess we could use the enum as type for those temps as well. Thoughts?
```suggestion
        PlayerActionInterruptResult interruptResult;
```